clanker-deployer/types.ts

```diff
@@ -0,0 +1,53 @@
+export type TokenPair =
+  | 'WETH'
+  | 'DEGEN'
+  | 'ANON'
+  | 'HIGHER'
+  | 'CLANKER'
+  | 'BTC'
+  | 'NATIVE'
+  | null;
+
+export interface IDeployFormData {
+  name: string;
+  symbol: string;
+  image?: File | null;
+  imageUrl?: string;
+  description?: string;
+  // Amount of paired token to use for the initial buy (creates initial liquidity)
+  devBuyAmount?: string | number;
+  // Percentage of token supply to lock in vesting contract (0-100)
+  lockupPercentage?: number;
+  // Unix timestamp when locked tokens can be released
+  vestingUnlockDate?: bigint | null;
+  telegramLink?: string;
+  websiteLink?: string;
+  xLink?: string;
+  marketCap?: string;
```

Comment:
where is this used? sorry i keep on asking and not seeing it

Reply:
this is for constructing the metadata in build-transaction